src/gui/sdlmain.cpp

```diff
@@ -4774,6 +4774,33 @@ int sdl_main(int argc, char* argv[])
 
 	const auto arguments = &control->arguments;
 
+	if (arguments->wait_pid) {
+		#ifdef WIN32
+		// Synchronize permission is all we need for WaitForSingleObject()
+		constexpr DWORD DesiredAccess = SYNCHRONIZE;
+		constexpr BOOL InheritHandles = FALSE;
+		HANDLE process = OpenProcess(DesiredAccess, InheritHandles, *arguments->wait_pid);
+		if (process) {
+			// Waits for the process to terminate.
+			// If we failed to open it, it's probably already dead.
+			WaitForSingleObject(process, INFINITE);
+			CloseHandle(process);
+		}
+		#else
+		while (1) {
```

Comment:
No love for `for (;;)`? 😎

Reply:
I don't really have a preference. I changed it for you :smile: